lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -672,15 +672,25 @@ def cmd_use(*args)
               mod = framework.modules.create(mod_name)
 
               unless mod
+                # Checks to see if we have any load_errors for the current module.
+                # and if so, returns them to the user.
+                load_error = framework.modules.load_error_by_name(mod_name)
+                if load_error
+                  print_error("Failed to load module: #{load_error}")
+                  return false
+                end
                 unless mod_resolved
+                  elog("Module #{mod_name} not found, and no loading errors found. If you're using a custom module" \
+                    ' refer to our wiki: https://github.com/rapid7/metasploit-framework/wiki/Running-Private-Modules')
+
                   # Avoid trying to use the search result if it exactly matches
                   # the module we were trying to load. The module cannot be
                   # loaded and searching isn't going to change that.
                   mods_found = cmd_search('-I', '-u', *args)
                 end
 
                 unless mods_found
-                  print_error("Failed to load module: #{mod_name}")
+                  print_error('Failed to load module: #{mod_name}')
```

Comment:
Single quotes will break string interpolation here

Reply:
Damn, I must have used the suggested changes by mistake. Will get that changed back.